docs/integrations/kafka.md

```diff
@@ -147,6 +147,8 @@ await getKafka().renameGroupId([
 ]);
 ```
 
+This will create the groupIds specified and will set them the offsets that the oldGroupId's had.
+If the new groupIds are found with offsets it does nothing.
```

Comment:
```suggestion
This will create the groupIds specified and will set the offsets that the oldGroupIds had.
If the new groupIds are found with offsets it does nothing.
```